### PR TITLE
Add eslint

### PR DESCRIPTION
**Pull request by xPaw**
Only basic config for now, more rules can be enabled in the future. I also added a CI job to run this.

### CODE REVIEW

File: code.user.js [hunk 1]
```diff
@@ -1,38 +1,42 @@
...
-// @downloadURL https://raw.githubusercontent.com/Nuklon/Steam-Economy-Enhancer/master/code.user.js
-// @updateURL   https://raw.githubusercontent.com/Nuklon/Steam-Economy-Enhancer/master/code.user.js
+// @name         Steam Economy Enhancer
+// @icon         https://upload.wikimedia.org/wikipedia/commons/8/83/Steam_icon_logo.svg
+// @namespace    https://github.com/Nuklon
+// @author       Nuklon
+// @license      MIT
+// @version      6.9.3
+// @description  Enhances the Steam Inventory and Steam Market.
+// @match        https://steamcommunity.com/id/*/inventory*
```
Comment (xPaw) [inline code.user.js:9]: I changed \@include to \@match as the userscripts lint suggested that for manifest v3.

File: code.user.js [hunk 2]
```diff
@@ -2209,8 +2212,8 @@
             var arr = [];
 
             for (var child in getActiveInventory().m_rgChildInventories) {
-                for (var key in getActiveInventory().m_rgChildInventories[child].m_rgAssets) {
-                    var value = getActiveInventory().m_rgChildInventories[child].m_rgAssets[key];
+                for (const key in getActiveInventory().m_rgChildInventories[child].m_rgAssets) {
```
Comment (xPaw) [inline code.user.js:2215]: Vars changed to const/let to reduce scope because they were redefining these.

**Comment by Nuklon** [comment]
Thanks for the PRs @xPaw. This now has a conflict, I think because of the match changes.

**Comment by xPaw** [comment]
@Nuklon rebased

**Comment by xPaw** [comment]
@Nuklon i've fixed that and opened a few more prs 😄

**Pull request merged**